lib/metanorma/collectionconfig/collectionconfig.rb

```diff
@@ -0,0 +1,238 @@
+require "fileutils"
+require "relaton"
+require "relaton-cli"
+require "yaml"
+require "shale"
+require "shale/adapter/nokogiri"
+require_relative "shale_monkeypatch"
+require_relative "helper"
+
+module CollectionConfig
+  ::Shale.xml_adapter = ::Shale::Adapter::Nokogiri
+
+  # TODO sidesteps bibdata redefinition of filter, will remove
+  class Manifest < ::Shale::Mapper
```

Comment:
This class needs to be called `module Metanorma; module CollectionConfig; class Manifest`.

The filename should be `lib/metanorma/collection_config/manifest.rb`.